modules/sequenzautils/bam2seqz/main.nf

```diff
@@ -38,11 +38,11 @@ process SEQUENZAUTILS_BAM2SEQZ {
         -t $tumourbam \\
         --fasta $fasta \\
         -gc $wigfile \\
-        -o ${prefix}.seqz.gz
+        -o ${prefix}.gz
 
     cat <<-END_VERSIONS > versions.yml
     ${getProcessName(task.process)}:
-        ${getSoftwareName(task.process)}: \$(sequenzautils --version 2>&1 | sed 's/^.*sequenzautils //; s/Using.*\$//')
+        ${getSoftwareName(task.process)}: \$(echo \$(sequenza-utils 2>&1) | sed 's/^.*is version //; s/ .*\$//')
```

Comment:
Missing `--version`? Or not required anymore?

Reply:
So why was it there first time :thinking: thats worrying :grimacing:. I wonder if it is possible to have pytest also spit out the contents of the `.yaml` so you can quickly check a version regex still works... (not just fail, but that it's actually formatting correctly)